app/src/test/kotlin/com/intive/picover/profile/viewmodel/ProfileViewModelTest.kt

```diff
@@ -65,20 +71,62 @@ class ProfileViewModelTest : ShouldSpec(
 			coVerify { authRepository.updateUserAvatar(uri) }
 		}
 
-		should("load profile WHEN updateAvatar called") {
-			coEvery { authRepository.updateUserAvatar(uri) } returns profile
-
-			tested.updateAvatar(uri)
+		should("load profile WHEN updateAvatar/updateName called") {
```

Comment:
the test name should be more generic, something like ..according to called method

Reply:
Done 😄